sys/xtimer/xtimer_core.c

```diff
@@ -147,18 +147,22 @@ static inline void _schedule_earliest_lltimer(uint32_t now)
 
 /**
  * @brief compare two timers. return true if timerA expires earlier than or equal to timerB and false otherwise.
+ *        assume that timerB starts later than timerA.
```

Comment:
Can this be safely assumed here in any case?
If not 100% sure, I'd prefer to add an explicit check for the start times to make it a proper compare function without any preconditions.

Reply:
Resolved.